src/query/algorithms/entityresolution/MinHash.cpp

```diff
@@ -0,0 +1,58 @@
+//
+// Created by root on 4/16/21.
+//
+
+#include "MinHash.hpp"
+
+arma::Col<short> MinHash::generateCRV(arma::Mat<float> &data, uint8_t d, bool quietPrint) {
+    display("Creating CRV", quietPrint);
+    //Calculate density vector
+    display("Create density vector", quietPrint);
+    arma::Col<float> denVec = getDensity(data);
+//        denVec.print();
```

Comment:
Lets remove the commented unused code segments

Reply:
Commented code removed